GoScrumApi/src/main/java/com/goscrum/GoScrumApi/controller/RoomController.java

```diff
@@ -0,0 +1,29 @@
+package com.GoScrum.GoScrumApi.controller;
+
+import com.GoScrum.GoScrumApi.payload.RoomDto;
+import com.GoScrum.GoScrumApi.service.RoomService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/rooms")
+public class RoomController {
+
+	private RoomService roomService;
+
+	public RoomController(RoomService roomService) {
+		this.roomService = roomService;
+	}
+
+	@PostMapping("/create")
+	@PreAuthorize("hasRole('ROLE_USER')")
+	public ResponseEntity<RoomDto> createRoom(@RequestBody RoomDto roomDto) {
```

Comment:
Nên validate the user input (sử dụng Spring Boot Validator)

Reply:
Se fix later, add @valid vao code bi loi, 

No validator could be found for constraint 'jakarta.validation.constraints.NotEmpty' validating type 'java.lang.Long'. Check configuration for 'hostId'